indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -175,10 +176,49 @@ ResultHeader.defaultProps = {
   categoryTitle: undefined,
 };
 
+function SearchOptions({sort, sortOptions, onSortChange}) {
+  if (!sortOptions.length) {
+    return null;
+  }
+
+  const selected = sortOptions.find(x => x.key === sort) || sortOptions[0];
+
+  return (
+    <div styleName="search-options">
+      <Dropdown
+        text={Translate.string('Sort by: {value}', {value: selected.label})}
+        onChange={(e, data) => onSortChange(data.value)}
+        options={sortOptions.map(x => ({value: x.key, text: x.label}))}
```

Comment:
```suggestion
        options={sortOptions.map(x => ({value: x.key, text: x.label}))}
        selectOnNavigation={false}
        selectOnBlur={false}
        value={selected.key}
```

This ensures the active option is shown in bold, and using the cursor keys without actively choosing an option won't immediately submit a query.